source/saml/tabs/add-app.rst

```diff
@@ -0,0 +1,25 @@
+Add a new SAML Application on GLPI
+----------------------------------
+
+First of all, you need to add a SAML application on GLPI because we need to report some information on GLPI <-> Entra.
+
+* In **Setup** > **SAML Providers**, click on **+ Add**

```

Comment:
The menu name **SAML Providers** has been aligned with the naming used by the Oauth SSO plugin. The menu button is currently: **SAML SSO Applications**

Reply:
The following is also important for security:

- They should not use the provided 'withlove' certificates and replace them with valid alternatives, there are no hard requirements to these certificates as fas as i am aware other then them being valid X509 certificates.
- For Entra the 'Required Authn' is adviced to be 'none' because any combination can be used with Microsoft.
- Security > Strict should be enabled in production environments 
- Security > JIT user creation must be on if they want to use the JIT rules. Matching is done against the glpi\User->'username' field. For the plugin to be able to authenticate a user, the field should contain a valid user principal name (UPN) formatted as an email. This behaviour might cause duplicate users in GLPI when users are migrating away from Ldap. This is an important detail because some users that are moving away from Active directory in some scenario's still use the usersamaccountname (old netbui names) as their UPN within entra. As a result the nameId field in the samlResponse will not be populated with an valid emailaddress. We use the username field because the email field is not guaranteed to be unique in GLPI and we need a unique identifier to allow authorization for a specific GLPI user.